src/components/Input/Input.tsx

```diff
@@ -37,13 +37,13 @@ export type InputProps = React.InputHTMLAttributes<HTMLInputElement> & {
    */
   isError?: boolean;
   /**
-   * Maximum number the input can take. When this number equals the input value, the plus button becomes disabled.
+   * Maximum number the input can take.
    */
-  max?: number;
+  max?: number | string | undefined;
```

Comment:
`?:` might handle the `undefined` case already

Reply:
I just copy pasted `number | string | undefined` from `type HTMLInputElement` without seeing the `?:` haha will just simplify to `number | string`